src/components/Navbar.js

```diff
@@ -12,43 +9,62 @@ const argumentsSet = [
   ["Investors", "For investors"]
 ];
 
+//WARNING: the wheel was invented here!!!
 class Navbar extends React.Component {
   constructor(props) {
     super(props);
     this.state = {activeLinkName: argumentsSet[0][0]};
   }
+  componentDidMount() {
+    window.onscroll = () => {
```

Comment:
use addEventListener instead of property
and unsubscribe listener in componentWillUnmount via removeEventListener